server/src/main/java/org/elasticsearch/client/internal/ClusterAdminClient.java

```diff
@@ -206,6 +206,10 @@ public ClusterStatsRequestBuilder prepareClusterStats() {
         return new ClusterStatsRequestBuilder(this);
     }
 
+    public ClusterStatsRequestBuilder prepareClusterStatsWithRemotes() {
```

Comment:
We'd really like to remove this `ClusterAdminClient` client interface - since the removal of the transport client it's only really used in tests. Definitely don't want to be adding to its API surface like this. Please just use `client.execute()` to invoke the request directly.